### PR TITLE
Wisconsin 2nd Tax Bracket (Single and HoH) 2023

### LINKED ISSUE #5029: Wisconsin 2nd Tax Bracket (Single and HoH) 2023
The 2nd tax bracket for single and HoH filers is set at 4% in PE when it should match the joint and separate filer rate at 4.4%. 

WI 2nd tax bracket

[Reference](https://www.revenue.wi.gov/TaxForms2023/2023-Form1-Inst.pdf)

### PR DESCRIPTION
**Pull request by hua7450**
Fixes #5029

### CODE REVIEW

File: policyengine_us/parameters/gov/states/wi/tax/income/rates/head_of_household.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ brackets:
       2023-01-01: 13_810
     rate:
       2021-01-01: 0.0465
-      2023-01-01: 0.04
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/wi/tax/income/rates/head_of_household.yaml:26]: could you add the 2023 tax form?

File: policyengine_us/parameters/gov/states/wi/tax/income/rates/separate.yaml [hunk 2]
```diff
@@ -10,7 +10,10 @@ metadata:
       href: https://www.efile.com/wisconsin-tax-brackets-rates-and-forms/
     - title: State of Wisconsin Department of Revenue
       href: https://www.revenue.wi.gov/Pages/FAQS/pcs-taxrates.aspx
-
+    - title: 2023 I-010 Form 1, Wisconsin income tax
+      href: https://www.revenue.wi.gov/TaxForms2023/2023-Form1f.pdf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/wi/tax/income/rates/separate.yaml:14]: Just the instructions should be enough, we only want forms that explicitly mention the rates and brackets

File: policyengine_us/parameters/gov/states/wi/tax/income/rates/head_of_household.yaml [hunk 3]
```diff
@@ -10,7 +10,8 @@ metadata:
       href: https://www.efile.com/wisconsin-tax-brackets-rates-and-forms/
     - title: State of Wisconsin Department of Revenue
       href: https://www.revenue.wi.gov/Pages/FAQS/pcs-taxrates.aspx
-
+    - title: 2023 I-010 Form 1 instructions
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/wi/tax/income/rates/head_of_household.yaml:13]:
```suggestion
    # The tax form only shows the tax brackets and 
    # not the exact rates
    - title: 2023 I-010 Form 1 instructions
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/wi/tax/income/rates/head_of_household.yaml:13]: all rate parameters

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**